certif/app/components/add-student-list.hbs

```diff
@@ -36,7 +36,7 @@
 
   {{#if this.hasCheckedSomething}}
     <div class="add-student-list__bottom-action-bar">
-      <button class="button button--link">Ajouter</button>
+      <button class="button button--link" {{on 'click' saveStudents}}>Ajouter</button>
```

Comment:
**Question:** On a _addStudents_, _enrolStudents_ and _saveStudents_, est-ce qu'il y a une difference entre ces concepts?

Reply:
Non c'est tous les même ! Il nous faudra une repasse à la fin pour harmoniser je pense